packages/web/src/common/store/pages/search-page/lineups/tracks/sagas.ts

```diff
@@ -43,13 +47,46 @@ function* getSearchPageResultsTracks({
       )
       results = tracks
     } else {
-      const { tracks } = yield* call(getSearchResults, {
-        searchText: query,
-        kind: category,
-        limit,
-        offset
-      })
-      results = tracks as unknown as Track[]
+      const getFeatureEnabled = yield* getContext('getFeatureEnabled')
+
+      const isSearchV2Enabled = yield* call(
+        getFeatureEnabled,
+        FeatureFlags.SEARCH_V2
+      )
+
+      if (isSearchV2Enabled) {
```

Comment:
i guess technically if this works, we dont need to check for the feature flag but the output is the same?

Reply:
Good point! I'll remove the branching